man/systemd.network.xml

```diff
@@ -997,6 +997,33 @@
         </varlistentry>
 
         <varlistentry>
+          <term><varname>RequestDHCPOptions=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of DHCP Parameter Request list option.
+            Defaults to none.</para>
```

Comment:
Let's make it more clear .  `RequestDHCPOptions=` Client will requests the server.